frontend/src/views/web/challenge-host-teams.html

```diff
@@ -33,7 +33,9 @@
                             </div>
                             <div class="collapsible-body">
                                 <h6><b>Team Members:</b></h6>
-                                <p ng-repeat="member in value.members">{{member.user}}</p>
+                                <p ng-repeat="member in value.members">{{member.user}}
+                                    <a class="show-member-title pointer" ng-if="value.created_by == web.name" ng-click="challengeHostTeams.confirmMemberDelete($event, value.id, member.id)"><i class="fa fa-trash list-icon text-highlight w-300 right"></i></a>
```

Comment:
Nice work with the `ng-if`. 
I am not familiar with frontend so I don't know if `web.name` is valid. 
Cloning and running it on my local is taking some time. Have you tried it on your side?

Reply:
> Nice work with the `ng-if`.
> I am not familiar with frontend so I don't know if `web.name` is valid.
> Cloning and running it on my local is taking some time. Have you tried it on your side?

yes @nikochiko . I will now update the GIF accordingly to show the changed behaviour.